R/ordering.R

```diff
@@ -82,3 +82,90 @@ setMethod("ordering<-", "DatasetCatalog", function (x, value) {
     }
     return(x)
 })
+
+#' Move a variable to after another variable
+#'
+#' @param x the variable group that `after` is in
+#' @param value the variable or dataset subset you would like to move
+#' @param after the variable you want to precede `value`
+#' @return returns a variable group
+#' @examples
+#' \dontrun{
+#' ordering(ds)[['Demographics']] <- moveToAfter(ordering(ds)[['Grp A']], ds['age'], ds$educ)
+#' }
+moveToAfter <- function(x, value, after){
```

Comment:
I'd just remove or comment this entirely if you're not really implementing/testing/exporting this yet.